### PR TITLE
Multiple state examples et al.

### PR DESCRIPTION
**Pull request by midlik**
# Description
I added a few examples for multi-state plus a few changes from our last meeting:
- change `transparency` to `opacity`
- allow `focus` node on `root`, add radius parameters
- make `linger_durantion_ms` required
- try to fix typing for mixins
- reorganize metadata for State vs States vs Snapshot

## Actions

- [x] Added description of changes to the `[Unreleased]` section of `CHANGELOG.md`
- [x] (Optional but encouraged) Added example(s) for new feature(s) in this PR to `app/api/examples.py`

### CODE REVIEW

**Review by sbittrich — APPROVED** [review]
LGTM, thanks.

**Comment by midlik** [comment]
Thank you Sebastian!
Please don't merge yet, there is one more thing that we changed with David. I need to test it and push.

File: molviewspec/molviewspec/builder.py [hunk 1]
```diff
@@ -720,14 +743,14 @@ def color(
         self._add_child(node)
         return self
 
-    def transparency(self, *, transparency: float = 0.8) -> Representation:
+    def opacity(self, *, opacity: float) -> Representation:
```
[the remark this answers is not in the document]
Comment (midlik) [reply molviewspec/molviewspec/builder.py:746]: It was kinda intentional, i didn't see a point in saying smth should be transparent but not specifying the exact opacity/transparency value.
But i can put it back if you insist.

[the remark this answers is not in the document]
Comment (sbittrich) [reply molviewspec/molviewspec/builder.py:746]: All good, makes sense to me.

**Comment by midlik** [comment]
OK, I added the metadata reorganization, so it should be ready now.

**Pull request merged**